src/main/java/com/kasperin/inventory_management/controllers/v1/ItemController.java

```diff
@@ -0,0 +1,33 @@
+package com.kasperin.inventory_management.controllers.v1;
+
+import com.kasperin.inventory_management.domain.FruitAndVege;
+import com.kasperin.inventory_management.domain.ProcessedFood;
+import com.kasperin.inventory_management.services.ItemService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping(ItemController.BASE_URL)
+public class ItemController {
+
+    public static final String BASE_URL = "/api/v1/items/";
+
+    private final ItemService itemService;
+
+    public ItemController(ItemService itemService) {
+        this.itemService = itemService;
+    }
+
+    @GetMapping
+    @ResponseStatus(HttpStatus.OK)
+    public Map<List<FruitAndVege>, List<ProcessedFood>> findAll(){
```

Comment:
Returning a map doesn't make sense here. I think you're getting ahead of yourself. You should be able to walk before you run. You would need to consolidate these types under a superclass, but I don't want this implemented until the other data yours are filled out and test coverage is in place for them.